VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/tasks-table/tasks-table.component.html

```diff
@@ -1,42 +1,55 @@
 <div class="task-header">
   <span class="task-header-body govuk-heading-s">Alerts for this hearing</span>
 </div>
-<div id="tasks-list" style="overflow: auto; height: xyz;" class="task-list">
+<div id="tasks-list" style="overflow: auto; height: xyz" class="task-list">
   <div *ngFor="let task of tasks" class="govuk-grid-row govuk-!-margin-0">
-    <div class="govuk-grid-column-one-quarter task-checkbox-and-time govuk-!-padding-right-0">
-      <div class="govuk-checkboxes__item">
-        <input
-          id="{{ task.id }}-checkbox"
-          class="govuk-checkboxes__input"
-          type="checkbox"
-          [checked]="task.status === 'Done'"
-          [disabled]="task.status === 'Done'"
-          (click)="completeTask(task)"
-        />
-        <label class="govuk-label govuk-checkboxes__label govuk-!-padding-left-2" for="task-created">
-          <p id="{{ task.id }}-created-date" class="govuk-body-s">{{ task.created | date: 'HH:mm' }}</p>
-        </label>
+    <ng-container
+      *ngIf="sessionStorage.get() && task.body !== 'Failed self-test (Incomplete Test)' && task.body !== 'Failed self-test (Bad Score)'"
+      ><ng-container *ngTemplateOutlet="alertTask"></ng-container
+    ></ng-container>
+
+    <ng-container *ngIf="!sessionStorage.get()"><ng-container *ngTemplateOutlet="alertTask"></ng-container></ng-container>
+
+    <ng-template #alertTask>
+      <div class="govuk-grid-column-one-quarter task-checkbox-and-time govuk-!-padding-right-0">
```

Comment:
It would be better to put an ngIf on this element

Reply:
Added

Reply:
lgtm! nice work